app/views/shared/_messages.html.slim

```diff
@@ -1,6 +1,7 @@
 - unless flash.empty?
   - flash.each do |name, msg|
     - if msg.present? && msg.is_a?(String)
-      .mxn2.sm-mxn6.sm-mtn3
-        div class="px2 py1 sm-px6 mb-40p h5 line-height-2 alert alert-#{name}" role='alert'
-          .alert-inner = msg
+      div class="alert-#{name}" role='alert'
+        div class="mxn1 mtn1 alert-icon"
+        div class="px1 py2 sm-px3 mb-50p h5 bold line-height-2 rounded alert-inner"
+          .alert-msg = msg
```

Comment:
i think we want 24px (1.5rem) spacing to the left and right of alert content.

i would do that like this:

the `p2` adds 16px on all sides, and then the added `px1` on the inner `alert-msg` adds the additional 8px required

also, i'm not sure we need the `mb-50p` class -- i think it should be 48px or `mb4` (which i added below)

```
        div class="p2 mb4 h5 bold line-height-2 rounded alert-inner"
          .px1.alert-msg = msg
```